importer/json_payloads/locations_payload.json

```diff
@@ -28,6 +28,15 @@
                         "display": "$t_display"
                     }
                 ]
+            },
+            {
+                "adminLevel": [
+                    {
+                        "system": "https://smartregister.org/codes/administrative-level",
+                        "code": "$adminLevelcode",
+                        "display": "$adminLevelcode"
```

Comment:
This display needs to have the string "Level" for it to show up as  `"display": "Level 3"`